athlinks-ontology.ttl

```diff
@@ -470,3 +467,14 @@ schema:GenderType dc:creator "nextdude"^^xsd:string ;
               )
 ] .
 
+## Properties/Predicates
+
+### http://adp.athlinks.com/schema/hasEntry
+:hasEntry a owl:ObjectProperty ;
```

Comment:
This is backwards I think. The whole point of creating Roles is to avoid polluting core classes like Person and Event with properties specific to our domain. I would just re-use the competitor property that currently has a domain of AthleticEvent (via inheritance from SportsEvent) and a range of Person or SportsTeam. I would also allow an AthleticEventEntry to be in the domain of competitor and keep its range as is.